app/models/user/reputation_tokens/arbitrary_token.rb

```diff
@@ -0,0 +1,22 @@
+class User::ReputationTokens::ArbitraryToken < User::ReputationToken
+  params :arbitrary_value, :arbitrary_reason
```

Comment:
As said, these have to be prefixed with `arbitrary_` due to conflicts with existing `value` and `reason` methods.